### PR TITLE
BAH-1113|BAH790|Buvaneswari|Distortion in the Manage Privilege page -…

### PR DESCRIPTION
**Pull request by buvaneswari-arun**
1. The css file causing the distortion is fixed.
2. UI improvements like appropriate message after saving is displayed
3. Fixing the form privilege grid to fetch the correct formprivileges

### CODE REVIEW

File: src/form-builder/components/FormPrivilegeTable.jsx [hunk 1]
```diff
@@ -216,40 +207,15 @@ export default class FormPrivilegeTable extends Component {
     }
     return null;
   }
-
-  _saveFormResource(formJson) {
```
Comment (angshu) [inline src/form-builder/components/FormPrivilegeTable.jsx:220]: @buvaneswari-arun why did you remove this method?

Comment (buvaneswari-arun) [reply src/form-builder/components/FormPrivilegeTable.jsx:220]: When this method is called here, then everytime a form privilege is saved, the formResource is also saved and an entry is made in the FormResource table. Now, I have restricted this updation to take place only when the whole form is saved and not everytime a privilege is saved. When the whole form is saved, then the privileges are gathered and written in the JSON

File: src/form-builder/components/FormDetailContainer.jsx [hunk 2]
```diff
@@ -177,11 +175,23 @@ export class FormDetailContainer extends Component {
         uuid: formResourceUuid,
       };
       this._saveFormResource(formResource);
-      this._saveFormPrivileges(
+
+      if (this.state.formPrivileges.length === 0) {
+        let version = this.state.formData.version;
+        version--;
```
Comment (binduak) [inline src/form-builder/components/FormDetailContainer.jsx:181]: @buvaneswari-arun can you please check the line formatting here

Comment (buvaneswari-arun) [reply src/form-builder/components/FormDetailContainer.jsx]: This piece is removed due to revamp of code.

**Review by angshu — APPROVED** [review]
